slidez-fe-core/src/containers/poll-editor/PollEditor.tsx

```diff
@@ -43,26 +43,72 @@ const PollEditor: React.FC<PollEditorProps> = ({ presentationId }) => {
 
     return (
         <div className='app'>
-            <h2 className='poll-name'>
-                <FontAwesomeIcon className='check-icon' icon={faCheck} />
-                Live poll
-            </h2>
+            <div>
+                <button
+                    className='back-button'
+                    onClick={() => alert('Not yet realized')}
+                >
+                    <svg
+                        width='24'
+                        height='24'
+                        viewBox='0 0 24 24'
+                        fill='#979797'
+                        xmlns='http://www.w3.org/2000/svg'
+                    >
+                        <g opacity='0.5'>
+                            <path
+                                fill='#979797'
+                                stroke='#979797'
+                                d='M15.1983 16.8019C15.3153 16.6848 15.3154 16.4951 15.1985 16.3779L11.0416 12.2119C10.9247 12.0948 10.9247 11.9052 11.0416 11.7881L15.1985 7.62213C15.3154 7.50494 15.3153 7.31517 15.1983 7.1981L14.2123 6.21213C14.0951 6.09497 13.9052 6.09497 13.788 6.21213L8.21229 11.7879C8.09513 11.905 8.09513 12.095 8.21229 12.2121L13.788 17.7879C13.9052 17.905 14.0951 17.905 14.2123 17.7879L15.1983 16.8019Z'
+                            />
+                        </g>
+                    </svg>
```

Comment:
Have a look how we insert svg as images and make the same (you can check dashboard sidebar)

Reply:
Done